ansible_collections/arista/avd/examples/dual-dc-l3ls/intended/structured_configs/dc1-leaf1c-debug-vars.yml

```diff
@@ -0,0 +1,6066 @@
+ansible_connection: ansible.netcommon.httpapi
```

Comment:
Is the presence of these debug-vars files in the PR intentional? The other examples don't have them as far as i can tell...

Reply:
please remove them. I also think we have some outdated ones on the l3ls